public/app/features/explore/Table.tsx

```diff
@@ -28,7 +28,7 @@ export default class Table extends PureComponent<TableProps> {
           if (link.className === 'link') {
             const columnKey = column.Header().props.title;
             const rowValue = rowInfo.row[columnKey];
-            this.props.onClickCell(columnKey, rowValue);
+            this.props.onClickCell?.(columnKey, rowValue);
```

Comment:
Same question whether this should not be non optional in props?

Reply:
Prop doesn't seem to be passed in [TablePanel](https://github.com/grafana/grafana/blob/9b7748ec133cb2ce4b092e523cb6d70a7b89de83/public/app/plugins/panel/table2/TablePanel.tsx#L26) so I think it's supposed to be optional?